6-expression.rst

```diff
@@ -1,4 +1,108 @@
+==========================================
 6. Analyzing RNAseq expression with Salmon
 ==========================================
 
-Salmon!
+We will use `salmon <http://salmon.readthedocs.org/en/latest/>`__ to
+quantify differential expression. `Salmon
```

Comment:
I think "quantify expression" would be more accurate than "quantify differential expression", no?